bin/configInitializer.js

```diff
@@ -0,0 +1,131 @@
+"use strict";
+
```

Comment:
This file should be in /lib and have the same header comments as all other files.